drivers/hmis/spec/models/hmis/hmis_data_cleanup_util_spec.rb

```diff
@@ -306,4 +309,84 @@ def expect_leaves_non_hmis_data_alone(&block)
       end
     end
   end
+
+  context 'with household ids duplicated across projects' do
+    before(:each) do
+      p2 = create(:hmis_hud_project, data_source: hmis_ds, organization: o1)
+      e1.update(household_id: 'ABCDEF', project: p1)
+      e2.update(household_id: 'ABCDEF', project: p2)
+      e3.update(household_id: 'ABCDEF', project: p2)
```

Comment:
probably want to get exception if this fails
```suggestion
      e1.update!(household_id: 'ABCDEF', project: p1)
      e2.update!(household_id: 'ABCDEF', project: p2)
      e3.update!(household_id: 'ABCDEF', project: p2)
```

Reply:
whoops thanks